### PR TITLE
Update README

### LINKED ISSUE #52: What is the problem with I2S sub-system on RPi OS with Linux kernel v5?
Excerpts from vocalfusion-rpi-setup README.md on 8 Dec 2022:

    We cannot use the latest as updates to linux kernel v5 have broken the I2S sub-system.

    **DO NOT** follow the prompt to update the software on the system. Set up the locale,
    and setup a network connect, but **DO NOT** update the software on the Raspberry Pi.
    This will update the kernel, and then the audio sub-system will not work.

I am running the latest Raspberry Pi OS 64-bit Bullseye 11.5 with Linux kernel version 5.15.76-v8+ on RPi 3 B+ connected to XVF3000 hardware, and audio (play and record) work properly, with good audio quality.

Is it surprising that audio works for my configuration?  What is the expected I2S problem that produced the README.md warning notes?  What should I look for to know if there is a problem lurking that I have not yet observed?

Thanks!

### PR DESCRIPTION
**Pull request by lucianomartin**
Closes #52

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,9 +48,7 @@ For XVF3510-UA and XVF361x-UA devices these actions will be done as well:
 
 ## Setup
 
-1. First, obtain the required version of the Raspberry Pi operating system, which is available here:
-
-   https://downloads.raspberrypi.org/raspbian/images/raspbian-2020-02-14/2020-02-13-raspbian-buster.zip
+1. First, obtain the latest version of the Raspberry Pi operating system, which can be installed with the Raspberry Pi Imager, available from [the Raspberry Pi website](https://www.raspberrypi.com/) at time of writing.
 
    We cannot use the latest as updates to linux kernel v5 have broken the I2S sub-system.
```
Comment (tomblackie-xmos) [inline README.md:53]: This is not correct now is it?  We are using the latest  version, and V5 (or V6 now) seems to work.

Comment (lucianomartin) [reply README.md]: I removed that line and the following ones.

File: README.md [hunk 2]
```diff
@@ -70,7 +65,7 @@ For XVF3510-UA and XVF361x-UA devices these actions will be done as well:
 
 2. Connect up the keyboard, mouse, speakers and display to the Raspberry Pi and power up the system. Refer to the **Getting Started Guide** for you platform.
 
-   **DO NOT** follow the prompt to update the software on the system. Set up the locale, and setup a network connect, but **DO NOT** update the software on the Raspberry Pi. This will update the kernel, and then the audio sub-system will not work.
+   Follow the prompt to update the software on the system. Set up the locale, and setup a network connect.
```
[the remark this answers is not in the document]
Comment (lucianomartin) [reply README.md]: I tried to run those commands on my older Pi and the I2S drivers are not compiling any more...

**Review by mbanth — APPROVED** [review]
The proposed changes look okay to me.

**Comment by lucianomartin** [comment]
Close this PR, as upgrading the Debian packages is not working on the latest 32-bit Raspbian image.

**Pull request closed without merging**